tokio/src/sync/mpsc/bounded.rs

```diff
@@ -68,6 +68,18 @@ pub struct Permit<'a, T> {
     chan: &'a chan::Tx<T, Semaphore>,
 }
 
+/// An [`Iterator`] of [`Permit`] that can be used to reserve `n` slots in the channel.
```

Comment:
This is inaccurate. You don't use it _to_ reserve slots in the channel. Rather, it holds slots that you've already reserved.

Reply:
"/// An [`Iterator`] of [`Permit`] that can be used to hold `n` slots in the channel."